core/storage/classifier/gae_models.py

```diff
@@ -190,8 +189,11 @@ def query_new_and_pending_training_jobs(cls, cursor=None):
                     datetime.datetime.utcnow())).order(
                         cls.next_scheduled_check_time, cls._key)
 
-        job_models, cursor, more = query.fetch_page(10, start_cursor=cursor)
-        return job_models, cursor, more
+        job_models = query.fetch(10, offset=offset)
+
+        offset = offset + len(job_models)
+
+        return job_models, offset
```

Comment:
The ``returns`` part of the docstring should be updated, too, given that the returned values are changing here.

Reply:
Done